parquet/src/test/java/org/apache/iceberg/parquet/TestBloomRowGroupFilter.java

```diff
@@ -193,6 +195,7 @@ public void createInputFile() throws IOException {
 
     // build struct field schema
     org.apache.avro.Schema structSchema = AvroSchemaUtil.convert(UNDERSCORE_STRUCT_FIELD_TYPE);
+    String compatibleFieldName = AvroSchemaUtil.makeCompatibleName("_incompatible-name");
```

Comment:
This is a personal nit of mine, but I don't like when the tests use the same implementation as the prod code to get expected values. Could we just hard wire in the converted name?